plugins/parsers/csv/parser_test.go

```diff
@@ -332,17 +332,40 @@ func TestSkipRows(t *testing.T) {
 			TimeFunc:          DefaultTime,
 		},
 	)
-	testCSV := `garbage nonsense
+	testCSV1 := `garbage nonsense
 line1,line2,line3
 hello,80,test_name2`
 
 	expectedFields := map[string]interface{}{
 		"line2": int64(80),
 	}
-	metrics, err := p.Parse([]byte(testCSV))
-	require.NoError(t, err)
-	require.Equal(t, "test_name2", metrics[0].Name())
-	require.Equal(t, expectedFields, metrics[0].Fields())
+	metrics0, err1 := p1.Parse([]byte(testCSV1))
+	require.NoError(t, err1)
+	require.Equal(t, "test_name2", metrics0[0].Name())
```

Comment:
Please don't add random suffices. Keep tests as they are and add new ones. This way we can be sure that the code is still working as before.